README.md

```diff
@@ -660,6 +661,25 @@ These are values that represent the current state of the downshift component.
 As a convenience, the `id` and `itemToString` props which you pass to
 `<Downshift />` are available here as well.
 
+## EventHandlers
+
+You can provide your own input handlers to Downshift which will be called before the default handlers.
+
+If you would like to prevent the default behavior, you can set the event's `preventDownshiftDefault` property to `false`.
+
+For example:
+
+```
```

Comment:
Let's add `javascript` here so we get syntax highlighting.

Also, could we add an example that shows that you can completely override the built-in prop by adding the prop after spreading the prop getter?